services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/models/AggregationMetadata.java

```diff
@@ -0,0 +1,13 @@
+package us.dot.its.jpo.ode.api.models;
+
+public class AggregationMetadata {
+    private long totalCount;
```

Comment:
suggestion(non-blocking): we can reduce the verbosity of the name with either of the following suggestions. `total` and `count` indicate the same things to me, so i don't know if they both need to be in the name
```suggestion
    private long count;
```
or
```suggestion
    private long total;
```

Reply:
updated